test/.ocamlformat-ignore

```diff
@@ -0,0 +1,2 @@
+#This is for tests only
```

Comment:
This file is part of the test code, but is in the top of the `test` dir, where currently we have only stuff to run the tests, etc. Maybe do the dir testing one level down using `test/passing/dir1` or something?